<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySQL StormCrow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/StormCrowIcon.png</normaloff>:/icon/StormCrowIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Connection</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Bases de donn√©es:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QListWidget" name="listWidgetDatabases">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Parametres de connexion:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Adresse du serveur:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditAdresse">
                 <property name="text">
                  <string>localhost</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Port:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditPort">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>3306</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Utilisateur:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditUser">
                 <property name="text">
                  <string>arnaud</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Mot de passe:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEditMDP">
                 <property name="text">
                  <string>ini01</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Connection</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>Se souvenir de moi</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tables et vues</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="1" column="0">
         <widget class="QToolBox" name="toolBoxTable">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="page_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>641</width>
             <height>288</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Tables et Vues</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0" colspan="2">
             <widget class="QListWidget" name="listWidgetTables">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_7">
           <attribute name="label">
            <string>Structure</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidgetTableDescription">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::NoSelection</enum>
              </property>
              <column>
               <property name="text">
                <string>Champs</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Nouvelle colonne</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Nouvelle colonne</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Nouvelle colonne</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Nouvelle colonne</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Type</string>
               </property>
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>641</width>
             <height>288</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Contenu</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="0" column="0">
             <widget class="QLabel" name="labelRappelTable">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTableView" name="tableViewContenu">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Requete</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QToolBox" name="toolBoxReq">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>641</width>
             <height>319</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Requete</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0" colspan="2">
             <widget class="QTextEdit" name="textEditReq">
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Entrez votre requete ici ....&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButtonExecReq">
              <property name="text">
               <string>Ex√©cuter</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>641</width>
             <height>319</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Affichage</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <widget class="QStackedWidget" name="stackedWidgetReq">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="page_8">
               <layout class="QGridLayout" name="gridLayout_12">
                <item row="0" column="0">
                 <widget class="QTableView" name="tableViewResultatReq"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_9">
               <layout class="QGridLayout" name="gridLayout_16">
                <item row="0" column="0">
                 <widget class="QTextEdit" name="textEditReqError">
                  <property name="styleSheet">
                   <string notr="true">color: rgb(170, 0, 0);</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Script</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QToolBox" name="toolBoxScript">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>641</width>
             <height>319</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Script</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="0" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLineEdit" name="lineEditAdresseScript">
                <property name="frame">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonScriptParcourir">
                <property name="text">
                 <string>Parcourir</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QTextEdit" name="textEditScript"/>
            </item>
            <item row="2" column="0">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>529</width>
                <height>24</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButtonExecScript">
              <property name="text">
               <string>Ex√©cuter</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>641</width>
             <height>319</height>
            </rect>
           </property>
           <attribute name="label">
            <string>R√©sultat</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0">
             <widget class="QTextEdit" name="textEditScriptResultat"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
